test/fixtures/golang/main.go

```diff
@@ -1,11 +1,13 @@
 package main
 
-import "io/ioutil"
-import "log"
-import "os"
+import (
+	"io"
+	"log"
+	"os"
+)
 
 func main() {
-	bytes, err := ioutil.ReadAll(os.Stdin)
+	bytes, err := io.ReadAll(os.Stdin)
```

Comment:
`ioutil.ReadAll` is depreacted from Go 1.16 and calls `io.ReadAll` instead. https://pkg.go.dev/io/ioutil#ReadAll